docs/adr/0003-content-organization-and-accesses.rst

```diff
@@ -0,0 +1,43 @@
+ADR 0003 - Content organization and accesses
+============================================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+We have `actors <0001-actors>`_, and `videos <0002-videos-languages>`_, that we'll call for now "content". We want this content to be organized for actors to manage/view them.
```

Comment:
Are "actors" really content? If by this you mean that actors can be managed by other actors, maybe explain it in a separate sentence and only call content what we imagine as content : video, audio and subtitle?

Reply:
rephrased (available in next git push)